source/loader/layers/sanitizer/linux/symbolizer.cpp

```diff
@@ -27,6 +29,20 @@ llvm::symbolize::PrinterConfig GetPrinterConfig() {
     return Config;
 }
 
+uptr GetModuleBase(const char *ModuleName) {
+    ModuleInfo Data{ModuleName, 0};
+    dl_iterate_phdr(
+        [](struct dl_phdr_info *Info, size_t, void *Arg) {
```

Comment:
It should be possible to just pass `ModuleName` into this lambda function instead of wrapping it into a struct.

Reply:
Thanks, done.